interapp-frontend/src/components/AnnouncementList/styles.css

```diff
@@ -0,0 +1,16 @@
+.announcement-list-container {
+    flex-direction: column;
+    justify-content: space-between;
+    padding: 1%;
+    /* setting rounded corners for the container*/
+    border-color: silver;
+    border-style:solid;
+    border-width: 5px;
+
+    border-radius: 15px;
+    background: linear-gradient(to bottom, #b4e391 50%,#6ad91a 100%);
```

Comment:
technically theres nothing wrong with this, but for consistency purposes could you use the mantine color variables? https://mantine.dev/theming/colors/
eg. ``var(--mantine-color-dark-9)``